pkg/crc/machine/kubeconfig.go

```diff
@@ -176,7 +176,7 @@ func addContext(cfg *api.Config, ip string, clusterConfig *types.ClusterConfig,
 	}
 
 	// append /clustername to AuthInfo
-	clusterUser, err := appendClusternameToUser(username, host)
+	clusterUser, err := appendClusternameToUser(username, clusterConfig.ClusterAPI)
```

Comment:
Can this be tested in kubeconfig_test.go somehow?

Reply:
will create a follow up PR to add this to the unit tests, i think refactoring `addContext` will be needed